demos/https/README.md

```diff
@@ -25,17 +25,17 @@
 ### Usage
 1. Run the presigned_url_gen.py with your s3 **bucket-name** and s3 object key **object-key**.  
    ```
-   python presigned_urls_gen.py --bucket my-bucket --key object-key
+   python presigned_urls_gen.py --bucket my_bucket --key object-key
```

Comment:
This needs to be something syntactically incorrect.  Anytime you provide shell commands, assume that someone will copy&paste them into a shell, e.g.:

```sh
    python presigned_urls_gen.py --bucket <YOUR BUCKET NAME> --key <YOUR OBJECT KEY> 
```

Reply:
Thanks